state-chain/pallets/cf-broadcast/src/tests.rs

```diff
@@ -58,29 +63,16 @@ impl MockCfe {
 		match event {
 			RuntimeEvent::Broadcaster(broadcast_event) => match broadcast_event {
 				BroadcastEvent::TransactionBroadcastRequest {
-					broadcast_attempt_id,
+					broadcast_id,
 					nominee,
 					transaction_payload: _,
 					transaction_out_id: _,
 				} => {
 					match scenario {
 						Scenario::SigningFailure => {
```

Comment:
we can rename this now to be `BroadcastFailure`

Reply:
Renamed